plugin/evm/vm.go

```diff
@@ -273,7 +274,7 @@ func (vm *VM) Initialize(
 		g.Config = params.SubnetEVMDefaultChainConfig
 	}
 
-	if g.Config.FeeConfig == (params.FeeConfig{}) {
+	if g.Config.FeeConfig == commontype.EmptyFeeConfig {
```

Comment:
Does this work with `big.Int` == 0 values? I would think they would need to be nil.

Reply:
no, but when genesis JSON is unmarshalled without the `FeeConfig`, it returns EmptyFeeConfig.